docs/01_general/01_schema-basics.md

```diff
@@ -101,24 +101,24 @@ GraphQL supports a few different types:
 
 <!--alex ignore-->
 
-Scalar types are similar to python primitive types. Here’s the list of the
+Scalar types are similar to Python primitive types. Here’s the list of the
 default scalar types in GraphQL:
 
-1. Int, a signed 32-bit integer, maps to python’s int
-2. Float, a signed double-precision floating-point value, maps to python’s float
-3. String, maps to python’s str
-4. Boolean, true or false, maps to python’s bool
-5. ID, a unique identifier that usually used to refetch an object or as the key
-   for a cache. Serialized as string and available as `strawberry.ID(“value”)`
+* Int, a signed 32-bit integer, maps to python’s int
+* Float, a signed double-precision floating-point value, maps to python’s float
+* String, maps to python’s str
+* Boolean, true or false, maps to python’s bool
+* ID, a unique identifier that usually used to refetch an object or as the key
+  for a cache. Serialized as string and available as `strawberry.ID(“value”)`
 
-> **NOTE:** strawberry also includes support for date, time and datetime
+> **NOTE:** Strawberry also includes support for date, time and datetime
 > objects, they are not officially included with the GraphQL spec, but they are
 > usually needed in most servers. They are serialized as ISO-8601.
 
 <!--alex ignore-->
 
 These primitives work for the majority for the uses cases, but you can also
```

Comment:
```suggestion
These primitives work for the majority of use cases, but you can also
```

Reply:
Almost there. `uses cases` -> `use cases`

Reply:
Damn, missed that